### DISCUSSION TITLE
Support MariaDB

### DISCUSSION THREAD

**Discussion by modem7** [diced/zipline#417] [Ideas]
Heya,

Would it be possible to support MariaDB at all?

Currently, all of my containers that require a DB use MariaDB, and it would make things more streamlined if it were possible to use this for zipline.

When attempting, I get:
```
error: Error validating datasource `db`: the URL must start with the protocol `postgresql://` or `postgres://`.
```

After searching issues like https://github.com/diced/zipline/issues/63 I thought it may have been possible.

**Comment by diced** [comment]
As said previously many times, Zipline will not support any other databases. Zipline is usually meant to be setup through docker, specifically docker compose. This makes it quite easy to spin up postgres containers that link with Zipline with no extra configuration needed from the user.
